src/posts/development_notam-2021-08.md

```diff
@@ -0,0 +1,117 @@
+- Rewritten ISIS.
+- Progress on VNAV and LNAV.
+- Experimental Version.
+
+---
+
+## Development Updates
+
+### Hydraulic Actuator
+
+Recently we showcased some fun flying with an open cargo door, but the actual point was to make a full simulation of moving parts, from rigid body physics to hydraulic actuator behaviour.
+
+In the following short video you can see our first try at a full physical based hydraulic simulation, using cargo door as our beloved guinea pig.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/KgzKziUz5bw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+You can first observe the unlock of the door by the ground crew, with actuators fighting the weight of the door. Then hydraulics powers the door, which is oscillating a bit with flow oscillations after initial pressure transient.
+
+We hope to port that initial implementation to all other moving part of the plane, so it can finally be alive!
+
+### Rewritten ISIS
+
+The Integrated Standby Instrument System (ISIS) has now been rewritten in React!
+
+In the spirit of making the A32NX the most well-equipped simulated aircraft, the ISIS now features a metric altimeter reading, as well as the barometric setting in both hPa and inHg providing the correct information for pilots all across the world! The baro knob can also be pushed to quickly convert to the STD pressure setting.
+
+The bugs page is also fully featured on the A32NX, allowing pilots to set relevant speeds and altitudes.
+
+Interestingly enough, airlines do not recommend setting bugs on the ISIS. This is because speed bugs previously set to take-off speeds would cause confusion on approach if the pilots were looking for characteristic approach speeds.
+
+We hope you don't have to stare at the ISIS too much because that would mean something has gone horribly wrong, but enjoy the capabilities of the little companion nonetheless!
+
+![ISIS Combined](/img/notam-images/isis-combined.jpg)
+
+### Custom VNAV Progress
+
+VNAV work has been steadily progressing, and builds upon the foundation set in place by our new custom FMS. Once completed, VNAV will introduce many long-awaited features to the A32NX, including accurate descent profile calculation, pseudo-waypoints (e.g. top of climb and top of descent), and accurate econ speed derivation. 
+
+Furthermore, since we are modeling the Honeywell Pegasus FMS2, we are able to model a unique feature no other Airbus addon is doing: CDAs, or continuous descent approaches. Rather than using level-altitude segments for deceleration during an approach, the Honeywell FMS instead builds the approach profile to decelerate WHILE descending, resulting in greater fuel savings. This will introduce two new pseudo-waypoints: FLAPS 1 and FLAPS 2. Here's a visualization (© Airbus Safety First 2021):
```

Comment:
Pegasus II is the hardware module

```suggestion
Furthermore, since we are modeling the Honeywell Release H3 FMS, we are able to model a unique feature no other Airbus addon is doing: CDAs, or continuous descent approaches. Rather than using level-altitude segments for deceleration during an approach, the Honeywell FMS instead builds the approach profile to decelerate WHILE descending, resulting in greater fuel savings. This will introduce two new pseudo-waypoints: FLAPS 1 and FLAPS 2. Here's a visualization (© Airbus Safety First 2021):
```